cmd/kubeadm/app/constants/constants.go

```diff
@@ -31,6 +31,9 @@ const (
 	APIServerKubeletClientCertName           = "apiserver-kubelet-client.crt"
 	APIServerKubeletClientKeyName            = "apiserver-kubelet-client.key"
 
+	ServiceAccountKeyBaseName = "sa"
+	ServiceAccountKeyName     = "sa.key"
```

Comment:
Got it now. Thanks for the explanation. Gonna take me some time to catch up and get familiar with everything. Appreciate all the help!